lib/lib.misc.php

```diff
@@ -1348,14 +1388,28 @@ function eventlog($type, $dataid, $action, $cid = null, $json = null, $id = null
 		return;
 	}
 
+	$idsCombined = implode(',', $ids);
```

Comment:
this assumes that no primary ID contains a comma - is that true for all primary IDs?

Reply:
For external primary ID's this might not be generally true, but in practice I think it is.

I have no easy solution if we do allow comma's, so we should think about this later on maybe?

Reply:
I would add a TODO so we don't forget about it.